quick-start/QuickStartTrackingApp/Views/TrackingView.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftUI
+
+struct TrackingView: View {
+    @ObservedObject var trackingViewModel: TrackingViewModel
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            if trackingViewModel.mapSigningIntialised {
+                MapView(trackingViewModel: trackingViewModel)
+                VStack {
+                    UserLocationView(trackingViewModel: trackingViewModel)
+                    CenterAddressView(trackingViewModel: trackingViewModel)
+                    TrackingBottomView(trackingViewModel: trackingViewModel)
+                }
+            }
+            else {
+                Text("Loading...") 
+            }
+        }
+        .onAppear() {
+            if !trackingViewModel.identityPoolId.isEmpty {
+                Task {
+                    do {
+                        try await trackingViewModel.authWithCognito(identityPoolId: trackingViewModel.identityPoolId)
+                    }
+                    catch {
+                        print(error)
```

Comment:
Should we log something here instead of printing?

Reply:
Added alert and logger